### PR TITLE
Issue #4748 symbolic links are deleted fix.

### PR DESCRIPTION
**Pull request by lastpeony**
https://github.com/ant-media/Ant-Media-Server/issues/4748

### CODE REVIEW

**Review by mekya — CHANGES_REQUESTED** [review]
@lastpeony thank you for your work.  
We need to have unit test code to merge this PR because we don't want to encounter the same problem again in the future.

**Comment by lastpeony** [comment]
> @lastpeony thank you for your work. We need to have unit test code to merge this PR because we don't want to encounter the same problem again in the future.

As i see unit test coverage is 100% on new code.
testSynchUserVoD in AntMediaApplicationAdapterUnitTest.java is covering it.

Did you mean functional test?

**Comment by mekya** [comment]
Yes, a kind of functional test because there is a bug in the code and we don't want to encounter the same problem again in the same way in the future. 

I've implemented the test code as a reference. 
I mean you can just check it and implement the same way for your future PRs

File: src/main/java/io/antmedia/AntMediaApplicationAdapter.java [hunk 1]
```diff
@@ -279,28 +278,32 @@ public boolean synchUserVoDFolder(String oldFolderPath, String vodFolderPath)
...
 		try {
 			if (!streamsFolder.exists()) {
 				streamsFolder.mkdirs();
 			}
-			if (vodFolder.exists() && vodFolder.isDirectory()) {
-				String newLinkPath = streamsFolder.getAbsolutePath() + File.separator + vodFolder.getName();
-				File newLinkFile = new File(newLinkPath);
+			if (vodFolder.exists() && vodFolder.isDirectory()) 
+			{
+				File newLinkFile = new File(streamsFolder, vodFolder.getName());
```
[the remark this answers is not in the document]
Comment (mekya) [reply src/main/java/io/antmedia/AntMediaApplicationAdapter.java:306]: It's an expected behavior.  @sonatype-lift ignore

[the remark this answers is not in the document]
Comment (mekya) [reply src/main/java/io/antmedia/AntMediaApplicationAdapter.java:306]: @sonatype-lift ignore

[the remark this answers is not in the document]
Comment (mekya) [reply src/main/java/io/antmedia/AntMediaApplicationAdapter.java:306]: @sonatype-lift ignoreall

**Review by mekya — APPROVED** [review]

**Pull request merged**